internal/credentials/kubernetes/ecr/pod_identity.go

```diff
@@ -157,8 +159,21 @@ func (p *podIdentityCredentialHelper) getAuthToken(
 	)
 	output, err := ecrSvc.GetAuthorizationToken(ctx, &ecr.GetAuthorizationTokenInput{})
 	if err != nil {
-		logger.Error(err, "error getting ECR authorization token")
-		return "", nil
+		var re *awshttp.ResponseError
+		if errors.As(err, &re) {
+			if re.HTTPStatusCode() != 403 {
+				return "", err
+			}
+			logger.Info(
```

Comment:
```suggestion
			logger.Debug(
```

This is likely to happen a lot and it's part of the normal course of business, so we should probably log it at a lower level.